core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -215,6 +216,7 @@ public synchronized void connect() throws AlluxioStatusException {
         mChannel = GrpcChannelBuilder
             .newBuilder(mAddress, mContext.getConf())
             .setSubject(mContext.getSubject())
+            .setExecutor(ForkJoinPool.commonPool())
```

Comment:
Is using the common pool the best choice for us?

Reply:
David made this the default pool for all clients. For worker client I changed it to the default gRPC cached thread pool. For master `AbstractClient` we can update to whatever we think is best for most scenarios.